cmd/setec/setec.go

```diff
@@ -147,20 +147,44 @@ generate the token, then re-run appending the provided value.`,
 }
 
 var serverArgs struct {
-	StateDir           string `flag:"state-dir,Server state directory"`
-	Hostname           string `flag:"hostname,Tailscale hostname to use"`
-	KMSKeyName         string `flag:"kms-key-name,Name of KMS key to use for database encryption"`
-	BackupBucket       string `flag:"backup-bucket,Name of AWS S3 bucket to use for database backups"`
-	BackupBucketRegion string `flag:"backup-bucket-region,AWS region of the backup S3 bucket"`
-	BackupRole         string `flag:"backup-role,Name of AWS IAM role to assume to write backups"`
-	Dev                bool   `flag:"dev,Run in developer mode"`
+	StateDir           string `flag:"state-dir,default='$STATE_DIR',Server state directory"`
+	Hostname           string `flag:"hostname,default='$HOSTNAME',Tailscale hostname to use"`
+	KMSKeyName         string `flag:"kms-key-name,default='$KMS_KEY_NAME',Name of KMS key to use for database encryption"`
+	BackupBucket       string `flag:"backup-bucket,default='$BACKUP_BUCKET',Name of AWS S3 bucket to use for database backups"`
+	BackupBucketRegion string `flag:"backup-bucket-region,default='$BACKUP_BUCKET_REGION',AWS region of the backup S3 bucket"`
+	BackupRole         string `flag:"backup-role,default='$BACKUP_ROLE',Name of AWS IAM role to assume to write backups"`
+	Dev                bool   `flag:"dev,default='$DEV',Run in developer mode"`
 }
 
 var clientArgs struct {
 	Server string `flag:"s,default=$SETEC_SERVER,Server address"`
 }
 
 func runServer(env *command.Env) error {
+	// If KMSKeyName is a filepath, read the file contents and use that as the key name. This allows users to specify the KMS key name via a file, which can be useful in some deployment scenarios.
+	if serverArgs.KMSKeyName != "" {
+		if data, err := os.ReadFile(serverArgs.KMSKeyName); err == nil {
+			serverArgs.KMSKeyName = strings.TrimSpace(string(data))
+		}
+	}
```

Comment:
I wasn't very consistent on this handling. There are several ideas here and I didn't see native support for it with the `flax` dependency